src/person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental
+    @rentals << rental
+  end
+
```

Comment:
- [ ] Great job by trying to implement the requested changes. But, in your implementation you missed something. In this method for adding rental the person itself should be sent as a parameter to create the new rental achieving the `'has-many'` association.
you can implement it as in the example below:
```ruby
 def add_rental(book, date)
     Rental.new(date, book, self)
  end
```